### PR TITLE
add astropy and scipy notebook and aux data

### PR DESCRIPTION
**Pull request by JouvinLea**
Here are the scipy and tutorial notebook.

I have an issue to load the video in the scipy notebook since tonight (-:! Can anyone just test localy if it works for him ?

### CODE REVIEW

File: astropy/solutions/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (vuillaut) [inline astropy/solutions/.idea/.gitignore:3]: I don't  think this file should be included

**Review by vuillaut — CHANGES_REQUESTED** [review]
I don't think the `.idea` folder belongs here

**Comment by vuillaut** [comment]
Could you also add two files (one for each course) in `docs/content/posts` following the `demo.md` so that the content of the classes appear in https://escape2020.github.io/school2022/ please?

**Comment by JouvinLea** [comment]
Do you have other comments @vuillaut ? otherwise you can merge.

**Comment by maxnoe** [comment]
I think there is some copy and paste stuff going on in the Readme, the scipy one has references to astropy

File: scipy/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Astropy
```
Comment (vuillaut) [inline scipy/README.md:1]: here

**Comment by maxnoe** [comment]
@JouvinLea your last commit says "clean" but it actually adds the output to the notebook. Did you forget to run "Cell -> Remove All Output"?

**Review by vuillaut — APPROVED** [review]

**Comment by vuillaut** [comment]
The `echo $HOME` won't work on windows but I think we can let this one go ;-)

**Comment by vuillaut** [comment]
The `astropy_hands_on` notebook doesn't run to the end

**Pull request merged**